File: Modul 1/C/PRAK110-2310817220025-FirdaKhoirunisa.c
#include<stdio.h>

void main(){
    //miring = 144-25
    int alas = 5;
    int tinggi = 12;
    int miring = 13;
    int keliling = (int) alas+tinggi+miring;
    int luas = (int) alas*tinggi/2;

    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", miring);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm", luas);
}

File: Modul 1/Python/PRAK110-2310817220025-FirdaKhoirunisa.py
import math
#Diketahui
alas = 5 
tinggi = 12
miring = math.sqrt(5**2 + 12**2)
Keliling = alas+tinggi+miring
Luas = alas*tinggi/2

print("Sisi A =", tinggi, ("cm"))
print("Sisi B =", int(miring), "cm")
print("Sisi C =", alas, "cm")
print("Keliling =", int(Keliling), "cm")
print("Luas", int(Luas), "cm")